test/unit_tests/database_test.dart

```diff
@@ -0,0 +1,32 @@
+void main() {
```

Comment:
dead code

Reply:
example for later db testing

Reply:
as u mentioned, we will add this feature later